pca_ica/run_pca.m

```diff
@@ -25,7 +27,7 @@ function run_pca(movie_source, varargin)
 medfilt_halfwidth = 1;
 autoset_num_PCs = 1;
 max_num_PCs = 1000;
-min_num_PCs = 300;
+min_num_PCs = 300; 

```

Comment:
I'm going to merge this for now, but stylistically, let's make sure there are no trailing whitespace in merged code.